src/Card.js

```diff
@@ -3,9 +3,17 @@ import PropTypes from 'prop-types'
 import ReactDOM, {unstable_renderSubtreeIntoContainer as renderSubtreeIntoContainer} from 'react-dom'
 import assign from 'object-assign'
 
-class Card extends React.Component {
-  static propTypes = {
+
+const defaultProps = {
+    active: false,
+    position: 'right',
+    arrow: null,
+    style: {style: {}, arrowStyle: {}}
+};
+
+const propTypes = {
```

Comment:
I liked the static props and defaultProps. What was the thought process behind doing it this way?

Reply:
It keeps the class clean,  declaring const propTypes and assign them later to the static properties. + it will be easy to reuse data structures across components.
defaultProps are const because they will not change through the program.